authors/apps/notifications/tests/test_create_and_fetch.py

```diff
@@ -0,0 +1,94 @@
+"""Module tests creation and fecthing of notifications"""
+from rest_framework.test import APIClient
+from django.urls import reverse
+from django.test import TestCase
+from rest_framework import status
+from ..models import Notification
+from ..models import Article
+
+
+class TestSetup(TestCase):
+    """Act as base class for all other test classes"""
+    def setUp(self):
+        """Initialize all testing data"""
+        self.client = APIClient()
+        #register a user to login with
+        self.user_data = {
+            "email": "miyagi@gmail.com",
+            "username": "miyagi",
+            "password": "miyagi123"
+        }
+        self.client.post(
+            'api/users/register',
+            self.user_data,
+            format="json")
+        #login the user
+        self.login_res = self.client.post(
+            'api/users/register', self.user_data, format="json")
+        self.user_token = self.login_res
+        #create notification object
+        self.notify_title = "Justo followed you"
+        self.notifications = Notification(title=self.notify_title)
+        self.token = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.\
```

Comment:
This is not a secure way to have a token displayed on the code.

Reply:
This is just a dummy token since there was no endpoint to generate real tokens, once the login endpoint is implemented I will just be getting it from the response.